zipline/assets/_assets.pyx

```diff
@@ -65,7 +67,9 @@ cdef class Asset:
                  object end_date=None,
                  object first_traded=None,
                  object auto_close_date=None,
-                 object exchange_full=None):
+                 object exchange_full=None,
+                 object tick_size="0.001",
```

Comment:
Yeah, I left it as a string primarily because I noticed its default value was an empty string. Similarly, it was defined as an `object` rather than a `float` in the cython definitions.